htsget.md

```diff
@@ -443,6 +445,124 @@ Initial guidelines, which we expect to revise in light of future experience:
+_optional boolean_
+</td><td>
+
+Indicates whether the service is capable of returning only a subset of available fields based on the `fields` query parameter.
+</td></tr>
+<tr markdown="block"><td>
+
+`tagsParametersEffective`  
+_optional boolean_  
+</td><td>
+
+Indicates whether the service is capable of server-side result tag inclusion/exclusion using the `tags` and `notags` query parameters.
+</td></tr>
+</table>
+
+Example service-info response:
+
+```
+{ 
+   "id":            "net.exampleco.htsget",
+   "name":          "ExampleCo genomics data service",
+   "version":       "0.1.0",
+   "organization":  {
+      "name":       "ExampleCo",
+      "url":        "https://exampleco.com"
+   },
+   "type":  {
+      "group":        "org.ga4gh",
+      "artifact":     "htsget",
+      "version":      "1.3.0"
+   },
+   "htsget": {
+      "datatype": "reads",
+      "formats":  ["BAM", "CRAM"],
+      "fieldsParameterEffective": true,
+      "tagsParametersEffective": false
+   }
+}
+```
+
+# GA4GH Service Registry
+
+The [GA4GH Service Registry API specification](https://github.com/ga4gh-discovery/ga4gh-service-registry) allows information about GA4GH-compliant web services, including htsget services, to be aggregated into registries and made available via a standard API. The following considerations SHOULD be followed when registering htsget services within a service registry.
+
+* Endpoints for different htsget data types should be registered as separate entities within the registry. If an htsget service provides both `reads` and `variants` data, both endpoints should be registered.
+* The `url` property should reference the API's ticket endpoint for a single data type, excluding any particular `id` (i.e. an htsget reads API registration should provide the URL to the reads ticket endpoint, an htsget variants API registration should provide the URL to the variants ticket endpoint). Clients should be able to assume that:
+   + Adding `/{id}` to the registered `url` will hit the corresponding ticket endpoint (i.e. `/reads/{id}` or `/variants/{id}`).
+   + Adding `/service-info` to the registered `url` will hit the corresponding `service-info` endpoint (i.e. `/reads/service-info` or `/variants/service-info`).
+* The value of the `type` object's `artifact` property should be `htsget` (i.e. the same as it appears in `service-info`)
+
+Example listing of htsget reads API and variants API registrations from a service registry's `/services` endpoint:
+
+```
+[
+   {
+      "id": "net.exampleco.htsget-reads",
+      "name": "ExampleCo htsget reads API",
+      "description": "Serves alignment data (BAM, CRAM) via htsget protocol",
+      "version": "0.1.0",
+      "url": "https://htsget.exampleco.com/v1/reads",
+      "organization":  {
+         "name":       "ExampleCo",
+         "url":        "https://exampleco.com"
+      },
+      "type": {
+         "group": "org.ga4gh",
+         "artifact": "htsget",
+         "version": "1.2.0"
```

Comment:
Will this (and the second one a dozen lines below) need to be bumped to `1.2.1`?